hack/test_e2e.sh

```diff
@@ -141,15 +141,26 @@ function setup_kind {
     # setup kind if not available in the system                               #
     #                                                                         #
     # Arguments:                                                              #
-    #   None                                                                  #
+    # arg1: installation dicrectory, path to where kind will be installed     #
     ###########################################################################
+    [ $# -eq 1 ]
+    if_error_exit "Wrong number of arguments to setup_kind"
+
+    local install_directory=$1
+
+    [ -d ${install_directory} ]
```

Comment:
Probably shellcheck tool will complain about this and recommend use quotes, example: "${install_directory}"
Could you please run the new script against: https://github.com/koalaman/shellcheck ?

Reply:
I will, I will update a push a new version tomorrow